.github/workflows/hassfest.yaml

```
@@ -1,14 +1,36 @@
   pull_request:
-  schedule:
-    - cron: "0 0 * * *"
+    branches:
+      - main
+
+permissions: {}
 
 jobs:
-  validate:
-    runs-on: "ubuntu-latest"
+  hassfest: # https://developers.home-assistant.io/blog/2020/04/16/hassfest
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)